queries/appSettings.ts

```diff
@@ -17,13 +16,7 @@ export async function getAppSetting<
   });
 
   if (!initializedAt) {
-    // initialize the app
-    // todo: should these come from fresco.config?
-    await createAppSetting('initializedAt', new Date());
-    await createAppSetting('configured', false);
-    await createAppSetting('allowAnonymousRecruitment', false);
-    await createAppSetting('limitInterviews', true);
-    await createAppSetting('installationId', createId());
+    await initializeWithDefaults();
```

Comment:
You could make this return the values it has just inserted, and then return the value here from this function, rather than then doing another query to fetch the value after you just set it.

Reply:
resolved [5739d8d](https://github.com/complexdatacollective/Fresco/pull/161/commits/5739d8d788ef202113dca7f45afbaf157db5ba79)